src/components/Contentful/CampaignLoanWrapper.vue

```diff
@@ -19,7 +19,7 @@
 
 				<campaign-loan-row
 					name="Loan Row"
-					v-if="componentProps.showLoanRows"
+					v-show="componentProps.showLoanRows"
```

Comment:
Just out of curiosity, how does this change help? Theoretically the component should be visible in the same circumstances before and after the change.